docs/source/api/exports.rst

```diff
@@ -15,10 +15,6 @@ Exports
     :members:
     :show-inheritance:
 
-.. autoclass:: TXTExports
-    :members:
-    :show-inheritance:
-
```

Comment:
Concerning docs, I [built ](https://txtexport-refactoring.readthedocs.io/en/latest/)docs from my distro without errors. Therefore, I don't understand what's the issue